GermanFilter/sections/specific.txt

```diff
@@ -116,6 +116,8 @@ netzwelt.de##.without-ad-label
 mtb-news.de##div[class*="header__banner"]
 radio.de##.topASp
 radio.de##.fevdIX
+urlaubspartner.net##a[target="_blank"][href^="https://cutt.ly/"]
+urlaubspartner.net##a[href^="href="https://firmen.faz.net/immobilienmakler"]
```

Comment:
It seems that this rule is not correct, I guess it should be:
`urlaubspartner.net##a[href^="https://firmen.faz.net/immobilienmakler"]`

Reply:
i was pretty sure i fixed this typo before pushing :(